docs/management.md

```diff
@@ -418,6 +446,66 @@ management.oauth_provider_url = https://my-web-portal
 With the previous settings, the management UI exposes the HTTP endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field. This is the endpoint where the Web portal will redirect users to the management UI.
 Additionally, RabbitMQ also accepts a JWT token in the HTTP `Authorization` header when the user lands on the management UI.
 
+With `sp_initiated` logon types, there is no need to configure the `oauth_provider_url` if `auth_oauth2.issuer` was set. However, for `idp_initiated` flows the `auth_oauth2.issuer` url may not necessarily be the url where to send users to authenticate. When this occurs, the `management.oauth_provider_url` overrides the `auth_oauth2.issuer` url.
+
+### Support multiple OAuth 2.0 resources {#support-multiple-resources}
+
+RabbitMQ 3.13 introduced support for multiple OAuth 2.0 resources in the OAuth 2.0 plugin and in the Management plugin.
+
+Once you have configured the [OAuth 2.0 plugin](./oauth2#multiple-resource-servers) with all the required OAuth 2.0 resources, they are automatically supported by the management UI.
+
```

Comment:
Is the above a prerequisite (do it at another time/previously) or is it the 1st step in this procedure? Please clarify which, thanks